Author: Pino Toscano <pino@debian.org>
Description: Add $$GMIC_LIB_PATH for zart
 Add a GMIC_LIB_PATH variable to the qmake build system of zart, so it
 is possible to specify where to find the dynamic gmic library, in the
 same way it is possible in gmic-qt.
 .
 If not specified, it falls back to $$GMIC_PATH, so behaving the same
 as before.
Last-Update: 2018-11-10
Forwarded: no

--- a/zart/zart.pro
+++ b/zart/zart.pro
@@ -43,6 +43,10 @@ defined(GMIC_PATH, var):!exists( $$GMIC_
 }
 message("G'MIC repository was found ("$$GMIC_PATH")")
 
+!defined(GMIC_LIB_PATH, var) {
+  GMIC_LIB_PATH = $$GMIC_PATH
+}
+
 unix {
    VERSION = $$system(grep \"define.ZART_VERSION \" include/Common.h | sed -e \"s/.*VERSION //\")
 }
@@ -78,7 +82,7 @@ openmp {
 
 equals(GMIC_DYNAMIC_LINKING, "on" ) {
   message(Dynamic linking with libgmic)
-  LIBS += $$GMIC_PATH/libgmic.so
+  LIBS += $$GMIC_LIB_PATH/libgmic.so
 }
 
 equals(GMIC_DYNAMIC_LINKING, "off" ) {
